pdns/dnsdist.hh

```diff
@@ -69,6 +73,7 @@ struct DNSQuestion
   DNSQuestion(const DNSQuestion&) = delete;
   DNSQuestion& operator=(const DNSQuestion&) = delete;
   DNSQuestion(DNSQuestion&&) = default;
+  virtual ~DNSQuestion() = default;
```

Comment:
Is this virtual dt needed? Only once class derives from `DNSQuestion` and it does not override the dt (and adds no fields either). Though I see it will make `dnsdist_ffi_dnsresponse_set_async` more complicated, as it wants to distinguish the two one way or another.